app/controllers/admin/api/credit_cards_controller.rb

```diff
@@ -15,8 +15,8 @@ class Admin::Api::CreditCardsController < Admin::Api::BaseController
   #
   ##~ op.parameters.add @parameter_access_token
   ##~ op.parameters.add @parameter_account_id_by_id
-  ##~ op.parameters.add :dataType => "string", :required => true, :paramType => "query", :name => "credit_card_token", :description => "Token used by the payment gateway to identify the buyer -- shopper reference (Adyen), customer profile ID (Authorize.net), customer ID (Braintree and Stripe), customer alias (Ogone-Ingenico). Some payment gateways may store more than one card under the same buyer reference and/or require an additional identifier for recurring payment. If you are using Braintree, there is no need for additional identifier -- the first credit card available will always be charged. For Adyen and Authorize.net, see `credit_card_authorize_net_payment_profile_token`."
-  ##~ op.parameters.add :dataType => "string", :required => false, :paramType => "query", :name => "credit_card_authorize_net_payment_profile_token", :description => "Additional reference provided by the payment gateway to identify a specific card under the same buyer reference. For Authorize.net, you MUST fill with the 'Payment profile token'. For Adyen, use the `recurringDetailReference` (provided  within the response for their `listRecurringDetails` API method), or leave it empty for always charging the LATEST card in their list. Not used for other payment gateways."
+  ##~ op.parameters.add :dataType => "string", :required => true, :paramType => "query", :name => "credit_card_token", :description => "Token used by the payment gateway to identify the buyer -- customer profile ID (Authorize.net), customer ID (Braintree and Stripe), customer alias (Ogone-Ingenico). Some payment gateways may store more than one card under the same buyer reference and/or require an additional identifier for recurring payment. If you are using Braintree, there is no need for additional identifier -- the first credit card available will always be charged. For Authorize.net, see `credit_card_authorize_net_payment_profile_token`."
```

Comment:
I think even if it is deprecated in the UI some customers on want to be able to set it through the API
Authorize.net is already deprecated but the documentation is still there. When we remove all deprecated gateway we can remove this documentation

@kevprice83 what do you think?

Reply:
hey @hallelujah. I think that returning it into the doc and remove after we completely clean the gateways makes total sense for me.

Reply:
I was not aware that this was even being deprecated. I think we need to consider who is using it and how they will migrate their customers from it to another payment gateway. Do we have a list of Providers already?

Reply:
@kevprice83 On SaaS only one provider. I will answer in the ticket ;)

Reply:
@hallelujah @kevprice83 I returned the Adyen into the doc. I think we can remove it from the doc only when we completely remove Adyen.